### PR TITLE
bpo-37363: Add audit events on startup for the run commands

### PR DESCRIPTION
**Pull request by zooba**
https://bugs.python.org/issue37363

### CODE REVIEW

**Comment by tiran** [comment]
Fantastic! This simplifies one of my PoCs for EP a lot.

Are these all entry points or is there another entry point for embedded Python interpreters?

**Comment by zooba** [comment]
There are definitely more entry points, but if someone else is embedding it then they have full control and audit hooks won't help :)

File: Doc/library/sys.rst [hunk 1]
```diff
@@ -905,6 +905,12 @@ always available.
    read, so that you can set this hook there.  The :mod:`site` module
    :ref:`sets this <rlcompleter-config>`.
 
+   .. audit-event:: cpython.run_interactivehook hook sys.__interactivehook__
```
Comment (tiran) [inline Doc/library/sys.rst:908]: I don't understand the audit-event syntax. The event takes one argument (hook), but the audit-event has two arguments. What's going on here?

Comment (zooba) [reply Doc/library/sys.rst:908]: I need to document the syntax somewhere...

Basically, it's:
```
.. audit-event:: event-name list,of,comma,separated,args anchor
```

Where the anchor is used for the backlink from the generated table (which appears to [not be generating right now](https://docs.python.org/3.8/library/audit_events.html)...).

**Comment by zooba** [comment]
I can't repro the Azure Pipelines failures locally (on Windows or Ubuntu), so I'm guessing it's environment related. Making one more guess before I start adding lots of diagnostic info :)

**Pull request merged**